src/router/routes.js

```diff
@@ -243,19 +243,17 @@ module.exports = [
 		},
 	},
 	{
+		name: 'lend-filter',
 		path: '/lend/filter',
-		component: () => import('@/pages/Lend/Filter/LendFilterPage'),
```

Comment:
At some point do we want to delete the old components?

Reply:
Yes we'll create some other tickets around the cleanup